docs/integration/alerts-signl4-integration.md

```diff
@@ -0,0 +1,30 @@
+title: Signl4 Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Signl4. Use them together to get the information about each notification quickly and on any device.
+
+## In Signl4
+
+**1.** Create a Signl4 account at [signl4.com](https://www.signl4.com).
```

Comment:
I'd zap this here and in all other hook pages

Reply:
Good point, I'll do that.